cellarium/ml/cli.py

```diff
@@ -309,6 +309,13 @@ def add_arguments_to_parser(self, parser: LightningArgumentParser) -> None:
                 }
             )
 
+        def predict(self, *args, **kwargs):
+            """Not well documented, but defining this here overrides the default predict subcommand.
+            This method injects return_predictions=False into the kwargs to prevent the predictions from
+            being returned, which prevents memory overflow when writing predictions to a file."""
+            kwargs["return_predictions"] = False
+            self.trainer.predict(*args, **kwargs)
+
```

Comment:
I think `return_predictions` can already be passed to the config file in default LightningCLI:

https://github.com/cellarium-ai/cellarium-ml/blob/090161e2baa1857d07850c7c18fea8dd09a056ef/tests/test_cli.py#L399

Reply:
I think the note about the `return_predictions` needs to be fixed because it is passed not to the init method (as it is implied by the note) but to the predict method via an additional argument in the config file:

```yaml
model:
  ...
data:
  ...
trainer:
  ...
return_predictions: false
```

Reply:
Aw man, okay, that's what I was missing

Reply:
I was trying to pass it from the config file, but had no idea where to put it and guessed all the wrong places.  How did you figure out that it goes there?  I spent quite some time reading docs... but never figured it out...

Reply:
It was the opposite experience for me. I initially generated a default predict config file with `... predict --print_config` which contained `return_predictions` and then had to figure out what it does from the documentation :)